### ISSUE TITLE
App bar menu buttons occasionally lose click event handlers

### ISSUE THREAD

**Issue by michenriksen** [michenriksen/gokiburi#2]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

After using Gokiburi for a bit, the top application bar buttons for pausing/resuming automatic test runs, running all tests, and opening settings occasionally seem to lose their `on:click` event handlers as they no longer react to clicks. 

A page refresh makes the buttons reactive again.

I suspect it could be a bug in [Skeleton's AppBar component](https://www.skeleton.dev/components/app-bar), but it needs more investigation.

### Expected Behavior

I expect the buttons to always be reactive to clicks without needing a page refresh.

### Steps To Reproduce

I haven't been able to find a way to consistently reproduce the bug, and I also haven't spotted any patterns in when it happens. It happens occasionally after Gokiburi has been running for a couple of minutes.

### Environment

```markdown
gokiburi:
  Version:    0.1.0
  Go Version: go1.20.4
  Git Commit: 2d2cbeeee11de49e533866010286a8f7f74311b1
  Released:   2023-05-07T15:12:28Z
  OS/Arch:    darwin/arm64

Browser: Firefox 112.0.2 / Google Chrome 112.0.5615.137
```

**Comment by vkorn** [comment]
I can consistently reproduce it: 
1. Refresh the page (re-run all tests)
2. Change something to make build fail
3. Click the refresh button -- this would fail the build
4. Refresh button stops working until the full refresh

```
2023/05/09 13:01:28 INFO runner: tests passed pass=true tests=64 packages=22 passed=64 skipped=0 failed=0 dur=4.362s  -- first refresh
2023/05/09 13:01:44 INFO runner: running tests... -- Updated func name to fail build/clicked refresh button
2023/05/09 13:01:46 ERRO gokiburi: test runner failed with error error="build failed"
```

**Comment by michenriksen** [comment]
@vkorn thanks for the reproduction steps, this was very helpful! 🙇 

Issue is fixed in [v0.1.3](https://github.com/michenriksen/gokiburi/releases/tag/v0.1.3).
